articles/man-in-the-middle-attack/index.md

```diff
@@ -0,0 +1,68 @@
+
+### Stages of Man-in-the-Middle Attacks
+#### Interception
+- **IP Spoofing** - Every computer on the internet has an internet protocol (IP) address. The attacker masks himself as an application and changes headers of the IP address. By spoofing an IP address, an attacker can trick you into thinking you’re interacting with a legitimate entity. Users attempting to access the URL of an application will be redirected to the attacker’s website instead. Thus, you will be giving the attacker access to the information you’d otherwise not share.
+- **DNS Spoofing** - The attacker accesses the server and alters the website address record to match his/her website records, thus redirecting the user through the attacker’s website. It’s a technique that forces a user to a fake website rather than the one that the user intends to visit. You may think you’re visiting a safe, trusted website when you’re actually interacting with a fraudster. The attacker’s goal is to divert traffic from the real site or capture user login credentials.
+- **ARP Spoofing** - Address Resolution Protocol (ARP) helps to translate IP addresses to physical Media Access Control (MAC) addresses which identify devices in a local area network. Hackers will respond to requests with their own MAC address using placed packets. This will direct the communication of the user to the hacker’s server where it can sniff personal information.
+
+#### Decryption
+- [HTTPS Spoofing](https://www.thewindowsclub.com/https-security-spoofing-man-in-the-middle/) - The attacker can send an SSL certificate containing the digital thumbprint of the user obtained from a compromised application. After that, the browser verifies it and approves, thus granting access to the attacker. The attacker fools your browser into believing it’s visiting a trusted website when it’s not. By redirecting your browser to an unsecured website, the attacker can check your interactions with that website and steal personal information you may be sharing.
+- [SSL Hijacking](http://techgenix.com/Understanding-Man-in-the-Middle-Attacks-ARP-Part4/) - An attacker passes forged authentication keys to both the user and application during a TCP handshake. This sets up what appears to be a secure connection when, in fact, the man in the middle controls the entire session.
+- [SSL Stripping](http://techgenix.com/Understanding-Man-in-the-Middle-Attacks-ARP-Part4/) - It minimizes an HTTPS association with HTTP by intercepting the TLS authentication sent from the application to the client. The attacker sends a decoded form of the application's site to the client while maintaining the anchored session with the application. In the meantime, the client's entire session is open to the attacker.
```

Comment:
`minimizes`, `associated`, `anchored session`. These feel like the wrong words. Wrong like a language translation tool might produce.

It might read better as the following, but still not sure if I correctly captured the intent.
```suggestion
- [SSL Stripping](http://techgenix.com/Understanding-Man-in-the-Middle-Attacks-ARP-Part4/) - It downgrades an HTTPS connection to HTTP by intercepting the TLS authentication sent from the application to the client. The attacker transfers an unencrypted form of the application's site to the client while maintaining the secure session with the application. In the meantime, the client's entire session is open to the attacker.
```